ff/src/fields/macros.rs

```diff
@@ -607,12 +607,11 @@ macro_rules! impl_Fp {
             }
         }
 
-        /// Outputs a string containing the value of `self`, chunked up into
-        /// 64-bit limbs.
         impl<P: $FpParameters> Display for $Fp<P> {
             #[inline]
             fn fmt(&self, f: &mut Formatter<'_>) -> FmtResult {
-                write!(f, stringify!($Fp"({})"), self.into_repr())
+                let value: num_bigint::BigUint = self.into_repr().into();
```

Comment:
I believe it should be safe to directly call `BigUint::from(self)`, as the `From`/`Into` impl already calls `into_repr()` https://github.com/arkworks-rs/algebra/blob/d7fa00e708d76465cc49ed5e33ea42ff17f599b1/ff/src/fields/macros.rs#L728

Reply:
Ah, yeah, it looks like the `into_repr()` could be skipped. Unfortunately it's not possible to call `BigUint::from`, since `Into` doesn't imply `From` (see #321, although this doesn't change the `Fp` conversions, only the `BigInteger` ones)

Reply:
If you'd like to add the `impl From<Fp*> for BigUint` in #321 that would be great. It probably makes sense to base this PR on top of #321 anyway due to changelog conflicts